distributed/client.py

```diff
@@ -1596,28 +1602,13 @@ def close(self, timeout=no_default):
             return
         self.status = "closing"
 
-        with suppress(AttributeError):
-            for pc in self._periodic_callbacks.values():
```

Comment:
calling PeriodicCallback.stop() calls TimerHandler.cancel which must be called on the eventloop thread